docs/shared/getting-started/start-new-project.md

```diff
@@ -0,0 +1,87 @@
+
+Choose a preset that matches your technology stack. This gives you a fully configured workspace.
+
+You can also choose **an empty workspace preset** which sets up the bare minimum configuration including Nx itself. This allows you to add technologies and features incrementally over time as you need them.
+
+<!-- ## Pick Your Stack!
+
+{% cards cols="3" lgCols="8" mdCols="6" smCols="5" moreLink="/showcase/example-repos" %}
+
+{% link-card title="Express" appearance="small" url="/technologies/node/express" icon="express" /%}
+{% link-card title="Vue" appearance="small" url="/technologies/vue/introduction" icon="vue" /%}
+{% link-card title="Next" appearance="small" url="/technologies/react/next" icon="nextjs" /%}
+{% link-card title="Nuxt" appearance="small" url="/technologies/vue/nuxt/introduction" icon="nuxt" /%}
+{% link-card title="Nest" appearance="small" url="/technologies/node/nest" icon="nestjs" /%}
+{% link-card title="Remix" appearance="small" url="/technologies/react/remix" icon="remix" /%}
+{% link-card title="Expo" appearance="small" url="/technologies/react/expo" icon="expo" /%}
+{% link-card title="React Native" appearance="small" url="/technologies/react/react-native" icon="react" /%}
+{% link-card title="Fastify" appearance="small" url="/showcase/example-repos/mongo-fastify" icon="fastify" /%}
+{% link-card title="Svelte" appearance="small" url="/showcase/example-repos/add-svelte" icon="svelte" /%}
+{% link-card title="Solid" appearance="small" url="/showcase/example-repos/add-solid" icon="solid" /%}
+{% link-card title="Lit" appearance="small" url="/showcase/example-repos/add-lit" icon="lit" /%}
+{% link-card title="Astro" appearance="small" url="/showcase/example-repos/add-astro" icon="astro" /%}
+{% link-card title="Qwik" appearance="small" url="/showcase/example-repos/add-qwik" icon="qwik" /%}
+
+{% link-card title="Rust" appearance="small" url="/showcase/example-repos/add-rust" icon="rust" /%}
+{% link-card title="Go" appearance="small" url="https://github.com/nrwl/nx-recipes/blob/main/go/README.md" icon="go" /%}
+{% link-card title=".NET" appearance="small" url="https://github.com/nrwl/nx-recipes/tree/main/dot-net-standalone" icon="dotnet" /%}
+{% link-card title="Cypress" appearance="small" url="/technologies/test-tools/cypress/introduction" icon="cypress" /%}
+{% link-card title="Playwright" appearance="small" url="/technologies/test-tools/playwright/introduction" icon="playwright" /%}
+{% link-card title="Vite" appearance="small" url="/technologies/build-tools/vite" icon="vite" /%}
+{% link-card title="Storybook" appearance="small" url="/technologies/test-tools/storybook" icon="storybook" /%}
+{% link-card title="Jest" appearance="small" url="/technologies/test-tools/jest/introduction" icon="jest" /%}
+{% link-card title="Rspack" appearance="small" url="/technologies/build-tools/rspack/introduction" icon="rspack" /%}
+
+{% /cards %} -->
```

Comment:
👍 I'm all down for removing/leaving commented out. I think we need to come up with a better way for showing and getting people to the areas of "I want to use X tool w/ Nx" instead of getting a giant list to pick from.